service/prometheus/scrapeconfig.go

```diff
@@ -183,6 +188,14 @@ func getScrapeConfigs(service v1.Service, certificateDirectory string) []config.
 				roleLabelRelabelConfig,
 				missingRoleLabelRelabelConfig,
 			},
+			MetricRelabelConfigs: []*config.RelabelConfig{
+				// keep only kube-system cadvisor metrics
```

Comment:
What about `kube-state-metrics`? There should also be a lot of metrics from other namespaces.

Reply:
`kube-state-metrics` are in a different Prometheus job, this is only affecting metrics scraped from  cadvisor endpoint